third_party/six.BUILD

```diff
@@ -1,6 +1,13 @@
+genrule(
+  name = "copy_six",
+  srcs = ["six-1.12.0/six.py"],
+  outs = ["__init__.py"],
+  cmd = "cp $< $(@)",
+)
+
 py_library(
-    name = "six",
-    srcs = ["six.py"],
-    srcs_version = "PY2AND3",
-    visibility = ["//visibility:public"],
+  name = "six",
+  srcs = ["__init__.py"],
```

Comment:
What are the advantages of this method over `strip_prefix`?

Reply:
Actually, there is an upcoming incompatible change that breaks with the `strip_prefix` version (Thanks to @aaliddell for pointing it out in https://github.com/protocolbuffers/protobuf/pull/6795#issuecomment-546060749).

> ... However, we can't know at the @six//:six py_library creation time what the legacy_create_init flag might be set to when anyone creates a py_binary that depends on six. Since a number of other repos depend on @six, we can't just fix all the py_binary definitions in protobuf. Therefore, by moving six.py to __init__.py, we force it to work regardless of the legacy_create_init flag, as it won't overwrite an existing __init__.py file.
>
> At some point legacy_create_init is getting flipped and removed, at which point this bodge can be fixed.